velox/dwio/parquet/tests/reader/ParquetTableScanTest.cpp

```diff
@@ -122,6 +124,27 @@ class ParquetTableScanTest : public HiveConnectorTestBase {
     assertQuery(plan, splits_, sql);
   }
 
+  void assertSelectWithTimezone(
+      std::vector<std::string>&& outputColumnNames,
+      const std::string& sql,
+      const std::string& sessionTimezone) {
+    auto rowType = getRowType(std::move(outputColumnNames));
+    auto plan = PlanBuilder().tableScan(rowType).planNode();
+    std::vector<exec::Split> splits;
+    splits.reserve(splits_.size());
+    for (const auto& connectorSplit : splits_) {
+      splits.emplace_back(folly::copy(connectorSplit), -1);
+    }
+
+    AssertQueryBuilder(plan, duckDbQueryRunner_)
+        .connectorSessionProperty(
+            kHiveConnectorId,
+            connector::hive::HiveConfig::kSessionTimezone,
```

Comment:
Do we know have 2 configs for the same value?  HiveConfig::kSessionTimezone and QueryConfig::kSessionTimezone . How will we make sure they are in sync?

Reply:
Hi @mbasmanova. Currently, there is no mechanism to guarantee that the two are completely consistent. I think we can use the same timezone when setting up the two configurations. The reason why we need a new HiveConfig is because the current connector cannot access the QueryConfig.

Reply:
@liujiayi771 This design feels error prone. Let's figure out a more robust solution. It appears that connectors do need access to some of the core configs. Timezone is just one of these. We don't want to expose all of engine configs to connectors, but looks like there is a need to expose a subset of well-curated configs. Let's do that.